File: no1/php/a.php
<?php

class Stack {

  public function __construct($list) {
    $this->stack = $list;
    $this->length = count($list);
  }

  public function empty_y(){
    if($this->length==0) return True;
    else return false;
  }

  public function T_peek(){
    return $this->stack[$this->length-1];
  }

  public function T_pop(){
    return $array_pop($this->stack);
  }

  public function T_push($value){
    array_push($this->stack, $value);
  }

  public function size(){
    return $this->length;
  }
}

?>

File: no1/a.py
# !/usr/bin/env python
# -*- coding: utf-8 -*-

class Stack:

    def __init__(stac):
        self.stac = stac

    def boolean empty():
        return True if len(stac)==0 else return False

    def T_peek():
        return stac(0)

    def T_pop():
        s = stac(0)
        stac.pop(0)
        return s

    def T_push(obj):
        stac.insert(0, obj)

    def size():
        return len(stac)
